pyxem/utils/io_utils.py

```diff
@@ -328,24 +345,31 @@ def _parse_hdr(fp):
 
 
 def _add_crosses(a):
-    """ Adds 3 pixel buffer cross to quad chip data.
+    """ Adds 3 pixel buffer cross to quad chip data. the input can be a stack or a reshaped 4DSTEM dask object. It returns
+    the object with the same number of axes.
```

Comment:
I'd revert this - it should be as concise as possible and the new information is contained in Parameters and Returns